src/platform/packages/shared/response-ops/rule_form/src/rule_form.scss

```diff
@@ -0,0 +1,27 @@
+.ruleForm__container {
+  container-type: inline-size;
+}
+
+.ruleFormFlyout__container {
+  container-type: inline-size;
+}
+
+@container (max-width: 768px) {
+  .euiDescribedFormGroup {
+    flex-direction: column;
+  }
+  .euiDescribedFormGroup > .euiFlexItem {
+    width: 100%;
+  }
+  .ruleDefinitionHeader {
+    flex-direction: column;
+    gap: 12px;
```

Comment:
Hey @Zacqary , please swap these four `px` values for EUI size and font size variables. Thanks

Reply:
Should be fixed